src/bezpath.rs

```diff
@@ -96,8 +115,36 @@ impl VelloVector for VelloBezPath {
 
         path
     }
+
+    fn border_translation(&self, time: f64) -> DVec2 {
+        let (path, t) = self.inbetween(time);
+
+        let current = path.0.end_point().unwrap_or_default();
+        let point = interp_pathel(current, path.1, t as f32)
+            .end_point()
+            .unwrap()
+            .to_vec2();
+
+        DVec2::new(point.x, point.y)
+    }
+
+    fn border_rotation(&self, time: f64) -> f64 {
```

Comment:
could optimise this with a simple match to check if were in a linear line or bezier curve and do easier calculations accordingly. usually we'll be in a curve though so it may be slower on average. what do you think?

Reply:
see https://github.com/voxell-tech/bevy_vello_graphics/pull/3#discussion_r1707025066 and [861dd7f](https://github.com/voxell-tech/bevy_vello_graphics/pull/3/commits/861dd7f657aaabe0aea7a6f39a464ef6cef48ca3)